modules/apps/29-fee/keeper/escrow_test.go

```diff
@@ -9,116 +9,6 @@ import (
-		receiveFee sdk.Coins
-		timeoutFee sdk.Coins
-		packetID   channeltypes.PacketId
-	)
-
-	testCases := []struct {
-		name     string
-		malleate func()
-		expPass  bool
-	}{
-		{
-			"success", func() {}, true,
-		},
-		{
-			"success with existing packet fee", func() {
-				fee := types.Fee{
-					RecvFee:    receiveFee,
-					AckFee:     ackFee,
-					TimeoutFee: timeoutFee,
-				}
-
-				packetFee := types.NewPacketFee(fee, refundAcc.String(), []string{})
-				feesInEscrow := types.NewPacketFees([]types.PacketFee{packetFee})
-
-				suite.chainA.GetSimApp().IBCFeeKeeper.SetFeesInEscrow(suite.chainA.GetContext(), packetID, feesInEscrow)
-			}, true,
-		},
-		{
-			"fee not enabled on this channel", func() {
-				packetID.ChannelId = "disabled_channel"
-			}, false,
-		},
-		{
-			"refundAcc does not exist", func() {
-				// this acc does not exist on chainA
-				refundAcc = suite.chainB.SenderAccount.GetAddress()
-			}, false,
-		},
-		{
-			"ackFee balance not found", func() {
-				ackFee = invalidCoins
-			}, false,
-		},
-		{
-			"receive balance not found", func() {
-				receiveFee = invalidCoins
-			}, false,
-		},
-		{
-			"timeout balance not found", func() {
-				timeoutFee = invalidCoins
-			}, false,
-		},
-	}
-
-	for _, tc := range testCases {
-		tc := tc
-
-		suite.Run(tc.name, func() {
-			suite.SetupTest()                   // reset
-			suite.coordinator.Setup(suite.path) // setup channel
-
-			// setup
-			refundAcc = suite.chainA.SenderAccount.GetAddress()
-			receiveFee = defaultReceiveFee
-			ackFee = defaultAckFee
-			timeoutFee = defaultTimeoutFee
-			packetID = channeltypes.NewPacketId(suite.path.EndpointA.ChannelConfig.PortID, suite.path.EndpointA.ChannelID, uint64(1))
-
-			tc.malleate()
-			fee := types.Fee{
-				RecvFee:    receiveFee,
-				AckFee:     ackFee,
-				TimeoutFee: timeoutFee,
-			}
-			packetFee := types.NewPacketFee(fee, refundAcc.String(), []string{})
-
-			// refundAcc balance before escrow
-			originalBal := suite.chainA.GetSimApp().BankKeeper.GetBalance(suite.chainA.GetContext(), refundAcc, sdk.DefaultBondDenom)
-
-			// escrow the packet fee
-			err = suite.chainA.GetSimApp().IBCFeeKeeper.EscrowPacketFee(suite.chainA.GetContext(), packetID, packetFee)
-
-			if tc.expPass {
-				feesInEscrow, found := suite.chainA.GetSimApp().IBCFeeKeeper.GetFeesInEscrow(suite.chainA.GetContext(), packetID)
-				suite.Require().True(found)
-				// check if the escrowed fee is set in state
-				suite.Require().True(feesInEscrow.PacketFees[0].Fee.AckFee.IsEqual(fee.AckFee))
-				suite.Require().True(feesInEscrow.PacketFees[0].Fee.RecvFee.IsEqual(fee.RecvFee))
-				suite.Require().True(feesInEscrow.PacketFees[0].Fee.TimeoutFee.IsEqual(fee.TimeoutFee))
-				// check if the fee is escrowed correctly
-				hasBalance := suite.chainA.GetSimApp().BankKeeper.HasBalance(suite.chainA.GetContext(), suite.chainA.GetSimApp().IBCFeeKeeper.GetFeeModuleAddress(), sdk.Coin{Denom: sdk.DefaultBondDenom, Amount: sdk.NewInt(600)})
```

Comment:
I see that we're no longer testing this logic anywhere now. I think we don't want to omit these checks :thinking: 

Kind of a pain in the ass to cover this but I think it's good to have. Maybe if it's not possible to do now we should take it as a sign to leave `EscrowPacketFee` public?

Reply:
I'll look into making assertions on balances in the msg_server tests 👍

Reply:
Done! ✅ 

Added assertions on escrow account balances for both success and failure cases!

Reply:
Actually, I just done it in the async handler. I'll push an update for the other one too!